### ISSUE TITLE
Attribute restrictions can be bypassed by swapping security token between entities

### ISSUE THREAD

**Issue by alexbudarov** [cuba-platform/cuba#879]
### Environment
- Platform version: 6.8 snapshot

### Description of the bug or enhancement
**TLDR**
Encrypted securityToken is not bound to particular entity instance. It can be copied from one entity and sent with another entity when sending information back to middleware. Thus security restrictions which rely on securityToken can be avoided.

Potentialy both Rest API and remote desktop client can exploit this. But it looks like Rest API does not use received securityToken for attribute restriction checks, so below case is only for altered desktop client.

**How to reproduce**
Apply attached patch for refapp 6.8.
[attribute-security.patch.txt](https://github.com/cuba-platform/cuba/files/2034100/attribute-security.patch.txt)

Patch changes:
- cuba.useSpringApplicationEventsToSetupAttributeAccess = false
- Add SetupAttributeAccessHandler for Model entity with logic:
> If Model manufacturer is "Toyota" then "Number of Seats" field is read-only.
- Change Model Editor:
  - you can edit or clear entity's security token.
  - clear readOnlyAttributes
  - set numberOfSeats field to be always editable

1) Launch refapp.
2) Create two Model entities:
- manufacturer = Lada, name = Vesta, numberOfSeats = 4
- manufacturer = Toyota, name = Corolla, numberOfSeats = 4

3) Check that attribute security works. You can edit "Number of seats" for Lada, but can't do it for Toyota.

4) Open Model Editor for Lada Vesta. Copy and remember its security token.

4) Open Model Editor for Toyota. Change "Number of Seats" = 1000. Set value of "Security token" field to previousy copied Lada's token. Press OK.

Actual: Changes are successfully saved.
Expected: numberOfSeats stays the same. Maybe even internal exception saying that "you're using the wrong token"?
